src/components/index-page/trusted-by/index.tsx

```diff
@@ -0,0 +1,146 @@
+
+import MetaLockup from "./logos/Meta.svg?svgr"
+import IBMWordmark from "./logos/IBM.svg?svgr"
+import AirBnBLockup from "./logos/AirBnB.svg?svgr"
+import IntuitWordmark from "./logos/Intuit.svg?svgr"
+import AWSLogo from "./logos/AWS.svg?svgr"
+import PayPalWordmark from "./logos/PayPal.svg?svgr"
+import NewYorkTimesWordmark from "./logos/NewYorkTimes.svg?svgr"
+import StarbucksWordmark from "./logos/Starbucks.svg?svgr"
+import ShopifyLockup from "./logos/Shopify.svg?svgr"
+import ShopifyMonotoneLockup from "./logos/ShopifyMonotone.svg?svgr"
+import GitHubLockup from "./logos/GitHub.svg?svgr"
+
+import styles from "./style.module.css"
+
+const logos: LogoListItem[] = [
+  {
+    href: "https://meta.com",
+    alt: "Meta",
+    component: MetaLockup,
+  },
+  {
+    href: "https://aws.amazon.com",
+    alt: "AWS",
+    component: props => (
+      <AWSLogo
+        {...props}
+        className={clsx(props.className, "h-[48px] sm:w-[110px]")}
+      />
+    ),
+  },
+  // todo: Netflix?
+  // {
+  //   href: "https://netflix.com",
+  //   alt: "Netflix",
+  //   component: NetflixWordmark,
+  // },
+  {
+    href: "https://airbnb.com",
+    alt: "Airbnb",
+    component: AirBnBLockup,
+  },
+  {
+    href: "https://intuit.com",
+    alt: "Intuit",
+    component: IntuitWordmark,
+  },
+
+  {
+    href: "https://ibm.com",
+    alt: "IBM",
+    component: props => (
+      <IBMWordmark {...props} className={clsx(props.className, "w-[140px]")} />
+    ),
+  },
+
+  {
+    href: "https://paypal.com",
+    alt: "PayPal",
+    component: PayPalWordmark,
+  },
+  {
+    href: "https://nytimes.com",
+    alt: "New York Times",
+    component: NewYorkTimesWordmark,
+  },
+  {
+    href: "https://starbucks.com",
+    alt: "Starbucks",
+    component: StarbucksWordmark,
+  },
+  {
+    href: "https://shopify.com",
+    alt: "Shopify",
+    component: ({ className, ...rest }) => (
+      <div
+        role="img"
+        className={clsx(styles.shopify, "relative size-full flex-col")}
+      >
+        <ShopifyLockup
+          className="size-full opacity-0 group-hover:opacity-100 group-focus:opacity-100"
+          {...rest}
+        />
+        <ShopifyMonotoneLockup
+          className="absolute inset-0 size-full opacity-100 group-hover:opacity-0 group-focus:opacity-0"
+          {...rest}
+        />
+      </div>
+    ),
+  },
+  {
+    href: "https://github.com",
+    alt: "GitHub",
+    component: GitHubLockup,
+  },
+]
+
+export function TrustedBy() {
+  return (
+    <section className="gql-section gql-container bg-neu-0 py-8 lg:py-16 xl:py-24">
+      <div className="flex flex-wrap justify-between gap-6">
+        <h2 className="typography-h3 max-w-[474px] text-neu-800">
+          GraphQL is open source and trusted by the industry
+        </h2>
+        <p className="typography-body-md max-w-[624px] text-neu-700">
+          Facebook's mobile apps have been powered by GraphQL since 2012. A
```

Comment:
It wasn't _a_ GraphQL spec, it was _the_ GraphQL spec - the one and only! :muscle: 

```suggestion
          Facebook's mobile apps have been powered by GraphQL since 2012. The
```